tests/conftest.py

```diff
@@ -827,6 +843,9 @@ def splinter_driver_kwargs(splinter_webdriver):
     """Disable certification verification for webdriver."""
     from selenium import webdriver
 
+    if not check_webdrivers():
+        pytest.skip('webdriver not found')
```

Comment:
How does this choose an available webdriver?

Reply:
Splinter does it for us. It chooses the driver based on availability. Should there be one more layer before it checks?

Reply:
It doesn't, which is why this PR was needed. It tries to use `firefox` even if there is no geckodriver binary.

Reply:
This is how you default to the driver specified on the CLI:

https://github.com/pytest-dev/pytest-splinter/blob/master/pytest_splinter/plugin.py#L95-L98